tests/tests/tier0/monitor-node-reconnect/python/is_node_connected.py

```diff
@@ -0,0 +1,22 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+import time
+import unittest
+
+from bluechi.api import Node
+
+
+class TestNodeIsConnected(unittest.TestCase):
+
+    def test_node_is_connected(self):
+        n = Node("node-foo")
+        assert n.status == "online"
```

Comment:
I am preparing a new test for `LogTarget` which can be useful to use this `test_node_is_connected` (using node name argument) is there a way we can re-use it?

Reply:
Very good question and I think that would be really valuable! 
In theory it is reusable, in practice we should move them to a central location, e.g. `tests/bluechi-scripts/`. I'll try to implement it and update this PR.